koku/api/report/queries.py

```diff
@@ -381,10 +383,12 @@ def get_resolution(self):
 
         if self.resolution == 'monthly':
             self.date_to_string = lambda dt: dt.strftime('%Y-%m')
+            self.string_to_date = lambda dt: datetime.datetime.strptime(dt, '%Y-%m').date()
```

Comment:
This needs to be a function pointer, not an actual function call. Just like the `self.date_to_string` above it.

Reply:
Can you give an example of what you mean?

Reply:
Taking a second look, I'm wrong. This is fine as-is. Sorry.